Sources/Fluent/Model/Model+CRUD.swift

```diff
@@ -49,7 +49,7 @@ extension Model where Database: QuerySupporting {
     ///     - conn: Database connection to use.
     /// - returns: Future that will be completed when the delete is done.
     public func delete(force: Bool = false, on conn: DatabaseConnectable) -> Future<Void> {
-        return Self.query(on: conn).delete(self, force: force)
+        return Self.query(on: conn, withSoftDeleted: true).delete(self, force: force)
```

Comment:
I think `withSoftDeleted` should only be true if `force` is true as well.

Reply:
I agree.  I'll add a case to the unit test and update the code.

Reply:
I've updated the code and extended the unit tests to cover the deletedAt property.